data/wanderer/wanderer missions.txt

```diff
@@ -0,0 +1,177 @@
+	destination "Sopi Lefarkata"
+	to offer
+		has "Wanderers: Mereti: The Plant 2: done"
+	on complete
+		conversation
+			`This time, Iriket guides you down into an equatorial lowland - a dry grassy plain over which a cool wind blows. You can feel the similarity to the steppe you visited on Solima Skarati, but this time you can breathe freely without the air stinging your eyes and mouth. Iriket takes some soil and places it in a small device, looking at it for a moment before turning to you. "Here. It will [grow, prosper] here."`
+			choice
+				`	"Is there anything else I can do to help?"`
+					goto nextsteps
+				`	"How can you be so sure?"`
+			`	"We have already collected significant data on the climate of this world. The only question I needed to answer in person was whether the [composition, nature] of the soil was suitable."`
+			label nextsteps
+			`	"Let us return to Vara Kehi'ki. We will send a small detachment of [healers, gardeners] to <planet>, to ensure that these plants become established. In the long term, we will return them to <origin> when it is healed."`
+
+mission "Wanderers: Mereti: The Plant 4"
+	landing
+	name "Return to <planet>"
+	passengers 1
+	description "Bring Iriket back to <destination>, in order to collect the Mereti plant and bring it to <origin>."
+	source "Sopi Lefarkata"
+	destination "Vara Kehi'ki"
+	to offer
+		has "Wanderers: Mereti: The Plant 3: done"
+	on complete
+		dialog `The Mereti drones from earlier are still waiting at the spaceport when you arrive. Iriket quickly disembarks and starts conversing with the locals, pausing only briefly to tell you to meet her in the spaceport.`
+
+mission "Wanderers: Mereti: The Plant 5"
+	name "Plant to <planet>"
+	description "Escort this Wanderer ship, carrying the plant and a team of Wanderers ready to help it grow, to <destination>."
+	source "Vara Kehi'ki"
+	destination "Sopi Lefarkata"
+	on visit
+		dialog `You have arrived on <planet>, but you have left the <npc> behind. It's carrying the plant, so you ought to take off and wait for it to arrive.`
+	to offer
+		has "Wanderers: Mereti: The Plant 4: done"
+	on offer
+		conversation
+			`You arrive in the spaceport just in time to see the plant being delicately loaded onto a Strong Wind. The Mereti drones sit stationary on the landing pad, and you get a distinct feeling that they are watching you intently.`
+			`	Iriket stands beside you as you watch, staring into the distance almost morosely, before speaking. "It is regrettable that we have been [forced, made] to use these ships for war. This is far closer to the purpose we originally [conceived, designed] them to [serve, fulfil]". You catch a glimpse of a feather falling to the ground as she speaks. Then, she interrupts herself. "No, I should not be so [disheartened?]. The Korath drones have turned their back on their creators' [wishes, design] and seek to [heal, tend] these worlds alongside us. That is something we can all appreciate."`
```

Comment:
```suggestion
			`	Iriket stands beside you as you watch, staring into the distance almost morosely, before speaking. "It is regrettable that we have been [forced, made] to use these ships for war. This is far closer to the purpose we originally [conceived, designed] them to [serve, fulfill]". You catch a glimpse of a feather falling to the ground as she speaks. Then, she interrupts herself. "No, I should not be so [disheartened?]. The Korath drones have turned their back on their creators' [wishes, design] and seek to [heal, tend] these worlds alongside us. That is something we can all appreciate."`
```
Actually, would it be possible to get the integration check to automatically make suggestions for spelling mistakes?